src/Radar.hpp

```diff
@@ -49,6 +51,21 @@ namespace radar_base {
          * zero
          */
         bool allZero() const;
+
+        /**
+         * @brief Normalize the sweep data of the radar in the speed map mode
+         *
+         * The normalized value is interpreted following these rules:
+         * 1 - Value 0 indicates that there is no object
+         * 2 - Value 127 indicates null speed
+         * 3 - Values above 127 indicates positive speeds, with increment of +0.25 m/s per
+         * unit. Eg. 130 represents +0.75 m/s
+         * 4 - Values below 127 indicates negative speeds, with increment of -0.25 m/s per
+         * unit. Eg. 125 represents -0.5 m/s
+         *
+         * @param speed_map_sweep_data The radar sweep data in speed map mode
+         */
+        static void normalizeSpeedMap(std::vector<uint8_t>& speed_map_sweep_data);
```

Comment:
Please also add some documentation for the radar speed map sweep data, this will help a lot when we get a different radar that we get to integrate into this.